dev-libs/libfmt/libfmt-3.0.1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+inherit cmake-utils
+
+DESCRIPTION="Formatting library for C++"
+HOMEPAGE="https://github.com/fmtlib/fmt"
+
+LICENSE="BSD-2"
+IUSE="test"
+SLOT="0"
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="git://github.com/fmtlib/fmt.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/fmtlib/fmt/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/fmt-${PV}"
+fi
+
+DEPEND=""
+RDEPEND=""
+
+src_configure() {
+	local mycmakeargs=(
+		-DFMT_TEST=$(usex test)
+		-DBUILD_SHARED_LIBS=ON
+	)
+	cmake-utils_src_configure
+}
+
+src_compile() {
+	cmake-utils_src_compile
+}
+
+src_install() {
+	cmake-utils_src_install
+}
+
+src_test() {
+	emake -C "${BUILD_DIR}" test
+}
```

Comment:
is this really needed? the default `cmake-utils` provided function did not suffice?
Also, the position of `src_test` is wrong here, it should come between `src_compile` and `src_install`